recipes-core/install-on-emmc/install-on-emmc_1.0.bb

```diff
@@ -11,7 +11,7 @@ inherit dpkg-raw
 DESCRIPTION = "This service provides the option to install on eMMC during first boot"
 
 DEPENDS = "regen-rootfs-uuid"
-DEBIAN_DEPENDS = "systemd, fdisk, util-linux, regen-rootfs-uuid"
+DEBIAN_DEPENDS = "systemd, fdisk, util-linux, regen-rootfs-uuid parted"
```

Comment:
Syntax error

Reply:
(it's a subtle bug I make too often as well, but it is quickly found when running the build)